src/mlpack/core/data/load_vec_impl.hpp

```diff
@@ -70,24 +68,22 @@ bool Load(const std::string& filename,
        * Now we can call the move operator, but it has to be the move operator
        * for Mat, not for Col.  This will avoid copying the data.
        */
-      *((arma::Mat<eT>*) &vec) = std::move(tmp);
+      *((arma::Mat<eT>*)&vec) = std::move(tmp);
       return true;
     }
   }
   else
   {
     // It's loaded as a column vector.  We can call the move constructor
     // directly.
-    *((arma::Mat<eT>*) &vec) = std::move(tmp);
+    *((arma::Mat<eT>*)&vec) = std::move(tmp);
```

Comment:
`SpaceAfterCStyleCast` handles this in clang-format. It has two values `true` or `false`. 

If `true`, a space is inserted after C style casts.
```c++
true:                                  false:
(int)i;                        vs.     (int) i;
```
I have set to `false` but it looks a bug in clang-format for dealing with such complex cases (not sure).